components/PaymentOutcome/index.tsx

```diff
@@ -16,13 +17,31 @@ interface Props {
 
 function PaymentOutcome({ paymentRequest, paymentAmount, dispatch }: Props) {
   const router = useRouter()
-  const { amount, unit, sats } = router.query
+  const { amount, unit, sats, username, memo } = router.query
   const { satsToUsd } = useSatPrice()
+  const [isPrint, setPrint] = React.useState(false)
```

Comment:
the setter and getter of the state should follow the same naming convention for easy code Readability
```suggestion
  const [isPrint, setIsPrint] = React.useState(false)
```